project/assets/evaluations.less

```diff
@@ -0,0 +1,85 @@
+
+body {
```

Comment:
General feedback: Please define a colors.less and use variables for all colors in the site instead of hardcoding them